can/interfaces/socketcan/constants.py

```diff
@@ -43,11 +43,6 @@
 MSK_ARBID = 0x1FFFFFFF
 MSK_FLAGS = 0xE0000000
 
-PF_CAN = 29
```

Comment:
python  has these integrated into the socket module. so these are a little redundant but I can leave them in if supporting before 3.3 is still required.

Reply:
The version currently developed on the `develop` branch (4.0.0-dev) only supports Python 3.6+. So it's safe to drop them.